components/automate-gateway/api/authz/pairs/pairs.go

```diff
@@ -8,7 +8,7 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
-	gwAuthzRes "github.com/chef/automate/components/automate-gateway/api/authz/response"
+	gwAuthzRes "github.com/chef/automate/components/automate-gateway/api/iam/v2/response"
```

Comment:
I was thinking we would move these files under `authz` under `iam` as well. otherwise i'm not sure what the distinction between `authz` and `iam` is at this point